### PR TITLE
Sagonzal/dev apps telemetry

### PR DESCRIPTION
**Pull request by sangonzal**
Instrumenting developer apps to send telemetry data to Aria. 

Two open items: 
1) Xforms.Android events are dropped before getting to the collector. Working with Aria team to resolve this.
2) Need to decide how we want to include the Microsoft.Applications.Events Nuget packages, as they are currently in a VSTS repository only accessible to Microsoft FTEs.

### CODE REVIEW

File: Telemetry/TelemetryServerReceiver.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    {
+        private ILogger logger;
+        private string msalEventNameKey;
+        private string ariaTenantId;
+
+        public TelemetryServerReceiver()
+        {
+            EVTStatus status;
+            LogManager.Start(new LogConfiguration());
+            logger = LogManager.GetLogger(ariaTenantId, out status);
```
Comment (henrik-me) [inline Telemetry/TelemetryServerReceiver.cs]: this seems to never be set? did you intend to use:  ariaTenantId = TelemetryReceiverConstants.AriaTenantId; ? #Closed

File: Telemetry/TelemetryClientReceiver.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+//
+//------------------------------------------------------------------------------
+
+extern alias Client;
+
+using System;
+using System.Collections.Generic;
+using Client::Microsoft.Applications.Events;
+
+namespace TelemetryReceivers
```
Comment (henrik-me) [inline Telemetry/TelemetryClientReceiver.cs]: We should probably re-consider the namespace:
perhaps: Microsoft.Identity.Client.Telemetry

**Review by henrik-me — CHANGES_REQUESTED** [review]
:clock1:

File: Telemetry/TelemetryServerReceiver.cs (hunk 1 above)
Comment (sangonzal) [reply Telemetry/TelemetryServerReceiver.cs]: Yes. Fixed it  #Closed

File: Telemetry/TelemetryClientReceiver.cs (hunk 2 above)
Comment (sangonzal) [reply Telemetry/TelemetryClientReceiver.cs]: Sure, changed it to Microsoft.Identity.Client.Telemetry

Comment (henrik-me) [reply Telemetry/TelemetryClientReceiver.cs]: I'm not seeing this change

---
In reply to: [234785661](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1412#discussion_r234785661) [](ancestors = 234785661)

Comment (sangonzal) [reply Telemetry/TelemetryClientReceiver.cs]: Should be there now. After the conversation offline, I also made changes to some of the other class names: changed namespace to Microsoft.Identity.Client.DevAppsTelemetry, and changed the "Receiver" classes to  ClientTelemetryHandler and ServerTelemetryHandler . I don't have a strong opinion either way, so if anybody has suggestions for better names I'm happy to change them.

**Comment by sangonzal** [comment]
Moved to MSAL repo

**Pull request closed without merging**